subprojects/gradle/module-dependencies-graph/src/gradleTest/kotlin/com/avito/module/dependencies/AppsDependenciesGraphTest.kt

```diff
@@ -1,163 +0,0 @@
-package com.avito.module.dependencies
-
-import com.avito.test.gradle.gradlew
-import org.junit.jupiter.api.BeforeEach
-import org.junit.jupiter.api.Test
-import org.junit.jupiter.api.io.TempDir
-import java.io.File
-
-// TODO: try to get rid of the printing task. It's used for testing only and fragile to changed output
-//  We can cover all cases by unit tests easier (see DependenciesGraphTest)
-internal class AppsDependenciesGraphTest {
```

Comment:
No need to support a task only for testing. 

The cases are covered by other tests:
- AndroidAppsGraphBuilderTest - unit tests for graph
- FindAndroidApp... - integration tests with Gradle Test Kit. They are real user scenarious.